jenkins/jobs/dsl/LoadTestCartridge.groovy

```diff
@@ -13,115 +13,279 @@ def loadCartridgeJob = freeStyleJob(projectFolderName + "/LoadDevCartridge")
+        environmentVariables {
+          env('PLUGGABLE_SCM_PROVIDER_PATH','${WORKSPACE}/job_dsl_additional_classpath/')
+          env('PLUGGABLE_SCM_PROVIDER_PROPERTIES_PATH','${WORKSPACE}/datastore/pluggable/scm')
+        }
+        groovy {
+          scriptSource {
+            stringScriptSource {
+              command('''
+import pluggable.scm.SCMProvider;
+import pluggable.scm.SCMProviderHandler;
+import pluggable.configuration.EnvVarProperty;
+
+EnvVarProperty envVarProperty = EnvVarProperty.getInstance();
+envVarProperty.setVariableBindings(System.getenv());
+
+String scmProviderId = envVarProperty.getProperty('SCM_PROVIDER_ID')
+
+SCMProvider scmProvider = SCMProviderHandler.getScmProvider(scmProviderId, System.getenv())
+
+def workspace = envVarProperty.getProperty('WORKSPACE')
+def projectFolderName = envVarProperty.getProperty('PROJECT_NAME')
+
+def cartridgeFolder = '';
+def scmNamespace = '';
+
+// Checking if the parameters have been set and they exist within the env properties
+if (envVarProperty.hasProperty('OVERWRITE_REPOS')){
+  overwriteRepos = envVarProperty.getProperty('OVERWRITE_REPOS')
+}else{
+  overwriteRepos = ''
+}
+if (envVarProperty.hasProperty('ENABLE_CODE_REVIEW')){
+  codeReviewEnabled = envVarProperty.getProperty('ENABLE_CODE_REVIEW')
+}else{
+  codeReviewEnabled = ''
+}
+if (envVarProperty.hasProperty('CARTRIDGE_FOLDER')){
+  cartridgeFolder = envVarProperty.getProperty('CARTRIDGE_FOLDER')
+}else{
+  cartridgeFolder = ''
+}
+if (envVarProperty.hasProperty('SCM_NAMESPACE')){
+  scmNamespace = envVarProperty.getProperty('SCM_NAMESPACE')
+}else{
+  scmNamespace = ''
+}
+
+String repoNamespace = null;
+
+if (scmNamespace != null && !scmNamespace.isEmpty()){
+  println("Custom SCM namespace specified...")
+  repoNamespace = scmNamespace
+} else {
+  println("Custom SCM namespace not specified, using default project namespace...")
+  if (cartridgeFolder == ""){
+    println("Folder name not specified...")
+    repoNamespace = projectFolderName
+  } else {
+    println("Folder name specified, changing project namespace value..")
+    repoNamespace = projectFolderName + "/" + cartridgeFolder
+  }
 }
+
+scmProvider.createScmRepos(workspace, repoNamespace, codeReviewEnabled, overwriteRepos)
 ''')
+                }
+            }
+            parameters("")
+            scriptParameters("")
+            properties("")
+            javaOpts("")
+            groovyName("ADOP Groovy")
+            classPath('''${WORKSPACE}/job_dsl_additional_classpath''')
+        }
+        environmentVariables {
+          env('PLUGGABLE_SCM_PROVIDER_PATH','${JENKINS_HOME}/userContent/job_dsl_additional_classpath/')
+          env('PLUGGABLE_SCM_PROVIDER_PROPERTIES_PATH','${JENKINS_HOME}/userContent/datastore/pluggable/scm')
+          propertiesFile('${WORKSPACE}/scm.properties')
+        }
         dsl {
-            external("cartridge/jenkins/jobs/dsl/**/*.groovy")
+            external("cartridge/**/jenkins/jobs/dsl/*.groovy")
```

Comment:
Doesn't losing the ** at the end mean that *.groovy files can't be in sub-directories any more or was that something we purposefully changed in the main Load_Cartridge job as this one is quite out of date?